AWSSDKSwiftCLI/Sources/SPR/Invalidate.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCLIUtils
+import AWSCloudFront
+
+extension SPRPublisher {
```

Comment:
Provides a function which is used to invalidate caching for the Registry version lists after they are updated.

On a spr-multi-publish run, only one invalidation is performed at the end, containing all version lists.  This is because Cloudfront will start to throttle validations if they are done one path at a time for each package.